docs/adr/0013-use-eclipse-orion-as-editor.md

```diff
@@ -0,0 +1,32 @@
+# Use Eclipse Orion as editor
+Editor should provide syntax highlighting and work in browser.
+
+## Considered Alternatives
+Compared the following cloud IDEs
+
+* Eclipse Che
+* Eclipse Orion
+* Codeanywhere
+* Cloud9
+* Codiad
+* Theia
+
+For more details see the [ultimate comparison]
```

Comment:
Please add the link directly after the brackets: `[...](...)`. Please add a final sentence dot.